autometrics/src/labels.rs

```diff
@@ -246,3 +228,75 @@ pub trait GetStaticStr {
     }
 }
 impl_trait_for_types!(GetStaticStr);
+
+#[macro_export]
+macro_rules! result_labels {
+    ($e:expr) => {{
+        use $crate::__private::{
+            GetLabels, GetStaticStr, ResultAndReturnTypeLabels, ERROR_KEY, OK_KEY,
+        };
+        $crate::__private::spez! {
```

Comment:
I think it would be better if we could call `spez` directly in the proc macro code and insert the token stream directly into the code the macro generates, instead of making it a dependency of the main `autometrics` crate and having the macro generate code that calls spez (unless there's a reason that cannot work)